MANIFEST.in

```diff
@@ -2,7 +2,7 @@ include LICENSE
 include AUTHORS
 include ChangeLog
 include requirements.txt
-include tendrl-glusterd.service
+include tendrl-gluster-integration.service
```

Comment:
I feel Tendrl:spec-28-logging-coverage branch is not upto date with the master . I was working on master . Should I send my changes against Tendrl:spec-28-logging-coverage ?